app/services/nomis/feature.rb

```diff
@@ -0,0 +1,28 @@
+module Nomis
+  class Feature
```

Comment:
Could this just be a module and be injected where needed (service mock spec helper, some views?) rather than having to type `Nomis::Feature` all the time?